desktop-app/app/main.dev.js

```diff
@@ -319,6 +321,103 @@ const createWindow = async () => {
     onResize();
   });
 
+  const getPermissionSettingPreference = () => {
```

Comment:
@jjavierdguezas Can you please move this function to `permissionUtils.js` and import from there?

Reply:
done